terraform/api_gateway.tf.json.template.py

```diff
@@ -619,6 +645,42 @@ def for_domain(cls, domain):
                         }
                     )
                 },
+                **(
+                    {
+                        'aws_ses_domain_identity': {
+                            'notify': {
+                                'domain': config.api_lambda_domain('notify')
+                            }
+                        },
+                        'aws_ses_identity_policy': {
+                            'notify': {
+                                'identity': '${aws_ses_domain_identity.notify.arn}',
+                                'name': config.qualified_resource_name('notify'),
+                                'policy': json.dumps({
+                                    'Version': '2012-10-17',
+                                    'Statement': [
+                                        {
+                                            'Effect': 'Allow',
+                                            'Principal': {
+                                                'AWS': 'arn:aws:sts::' + aws.account
+                                                       + '/'.join([':assumed-role',
+                                                                   config.qualified_resource_name(app.name),
+                                                                   config.qualified_resource_name(app.name,
+                                                                                                  '-notify')
```

Comment:
WIP: Is this preferred over the following?
```suggestion
                                                                   config.indexer_name,
                                                                   config.indexer_function_name('notify')

```